### PR TITLE
[MRG] Integration - teaser API doc

### PR DESCRIPTION
**Pull request by michelfernandez**
- Add API-Doc in Footer as a link
- Add Teaser in Integration for API-Doc
- Changed style for feature titles to 20px

### CODE REVIEW

File: site/content/english/integration.md [hunk 1]
```diff
@@ -85,6 +85,39 @@ With Skribble, you choose a cloud platform that combines <br class="hide-for-mob
 
 [//]: # (--------------------------------------------------------------------------------------------------------------)
 
+{{% content class="no-bottom-padding" %}}
+{{% heading %}}
+# Try out our API
```
Comment (Kirastina) [inline site/content/english/integration.md]: Try out the API

File: site/content/english/integration.md [hunk 2]
```diff
@@ -85,6 +85,39 @@ With Skribble, you choose a cloud platform that combines <br class="hide-for-mob
...
+When you sign up for the Skribble business account, <br class="hide-for-mobile">you can create demo or productive API keys.
```
Comment (Kirastina) [inline site/content/english/integration.md]: You can create a demo key in your Skribble Business account.

File: site/content/french/integration.md [hunk 3]
```diff
@@ -85,6 +86,36 @@ Avec Skribble, vous choisissez une plate-forme de cloud <br class="hide-for-mobi
 
 [//]: # (--------------------------------------------------------------------------------------------------------------)
 
+{{% content class="no-bottom-padding" %}}
+{{% heading %}}
+# Essayez notre API
```
Comment (Kirastina) [inline site/content/french/integration.md]: Essayez l'API

File: site/content/french/integration.md [hunk 4]
```diff
@@ -85,6 +86,36 @@ Avec Skribble, vous choisissez une plate-forme de cloud <br class="hide-for-mobi
...
+When you sign up for the Skribble business account, <br class="hide-for-mobile">you can create demo or productive API keys.
```
Comment (Kirastina) [inline site/content/french/integration.md]: Vous pouvez créer une clé de démonstration dans votre compte Skribble Business.

File: site/content/german/integration.md [hunk 5]
```diff
@@ -83,6 +84,37 @@ Mit Skribble wählst du eine Cloud-Plattform, die technologische Sicherheit <br
 
 {{< /features-container >}}
 
+{{% content class="no-bottom-padding" %}}
+{{% heading %}}
+# Probiere unsere API aus
```
Comment (Kirastina) [inline site/content/german/integration.md]: API ausprobieren

File: site/content/german/integration.md [hunk 6]
```diff
@@ -83,6 +84,37 @@ Mit Skribble wählst du eine Cloud-Plattform, die technologische Sicherheit <br
...
+Wenn Du Dich mit einem Business Account registrierst, <br class="hide-for-mobile">kannst du sofort API-Key's erzeugen – für den Demo- oder Produktiven-Betrieb. <br class="hide-for-mobile">
```
Comment (Kirastina) [inline site/content/german/integration.md]: In deinem Skribble Business Konto kannst du einen Demo-Schlüssel erstellen.

File: site/content/french/integration.md [hunk 7]
```diff
@@ -85,6 +86,36 @@ Avec Skribble, vous choisissez une plate-forme de cloud <br class="hide-for-mobi
...
+{{< picture api-main 600 "Un ordinateur portable avec des panneaux de remplissage et une roue dentée." >}}
+
+{{% heading %}}
+{{% /heading %}}
+
+[//]: # (--------------------------------------------------------------------------------------------------------------)
+
+{{< cta-group >}}
+{{% cta
+  label="API documentation"
```
Comment (Kirastina) [inline site/content/french/integration.md]: Documentation API

File: site/content/english/integration.md [hunk 8]
```diff
@@ -85,6 +85,39 @@ With Skribble, you choose a cloud platform that combines <br class="hide-for-mob
...
+{{< picture api-main 600 "A laptop with fountain pen and gear wheel." >}}
+
+{{% heading %}}
+{{% /heading %}}
+
+[//]: # (--------------------------------------------------------------------------------------------------------------)
+
+{{< cta-group >}}
+{{% cta
+  label="API Dokumentation"
```
Comment (Kirastina) [inline site/content/english/integration.md]: API documentation

File: site/content/german/integration.md [hunk 9]
```diff
@@ -83,6 +84,37 @@ Mit Skribble wählst du eine Cloud-Plattform, die technologische Sicherheit <br
...
+{{< picture api-main 600 "Ein Laptop mit Füllfeder und Zahnrad." >}}
+
+{{% heading %}}
+{{% /heading %}}
+
+[//]: # (--------------------------------------------------------------------------------------------------------------)
+
+{{< cta-group >}}
+{{% cta
+  label="API Dokumentation"
```
Comment (Kirastina) [inline site/content/german/integration.md]: API-Dokumentation

File: site/content/english/integration.md (hunk 1 above)
Comment (michelfernandez) [reply site/content/english/integration.md]: Corrected

File: site/content/english/integration.md (hunk 2 above)
Comment (michelfernandez) [reply site/content/english/integration.md]: Corrected

File: site/content/french/integration.md (hunk 3 above)
Comment (michelfernandez) [reply site/content/french/integration.md]: Corrected

File: site/content/french/integration.md (hunk 4 above)
Comment (michelfernandez) [reply site/content/french/integration.md]: Corrected

File: site/content/german/integration.md (hunk 5 above)
Comment (michelfernandez) [reply site/content/german/integration.md]: Corrected

File: site/content/german/integration.md (hunk 6 above)
Comment (michelfernandez) [reply site/content/german/integration.md]: Corrected

File: site/content/french/integration.md [hunk 10]
```diff
@@ -85,6 +86,36 @@ Avec Skribble, vous choisissez une plate-forme de cloud <br class="hide-for-mobi
...
+  href="https://doc.skribble.com/"
+  outlined="true"
+  target="_blank"
+%}}
+
+{{< picture "api-dok" 140 "Une fenêtre d'interface utilisateur avec un contrat et un engrenage" >}}
+
+{{% richtext %}}
+**Documentation API**<br>
+Consultez notre documentation API
```
[the remark this answers is not in the document]
Comment (michelfernandez) [reply site/content/french/integration.md]: Corrected

File: site/content/german/integration.md [hunk 11]
```diff
@@ -83,6 +84,37 @@ Mit Skribble wählst du eine Cloud-Plattform, die technologische Sicherheit <br
...
+  href="https://doc.skribble.com/"
+  outlined="true"
+  target="_blank"
+%}}
+
+{{< picture "api-dok" 140 "Userinterface Fenster mit Zahnrad" >}}
+
+{{% richtext %}}
+**API-Dokumentation**<br>
+Teste jetzt unsere API.
```
[the remark this answers is not in the document]
Comment (michelfernandez) [reply site/content/german/integration.md]: Corrected

File: site/content/french/integration.md (hunk 7 above)
Comment (michelfernandez) [reply site/content/french/integration.md]: Corrected

File: site/content/english/integration.md (hunk 8 above)
Comment (michelfernandez) [reply site/content/english/integration.md]: Corrected

File: site/content/german/integration.md (hunk 9 above)
Comment (michelfernandez) [reply site/content/german/integration.md]: Corrected

**Comment by betatim** [comment]
The language of the testimonials is mixed up. Michel is fixing it after which we can merge this.

**Pull request merged**